app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require bootstrap-sprockets
-//= require_tree .
```

Comment:
Pay attention here. If it removed, next time everyone should import which they want manually. But it is also a good idea as long as we notice it.